crate/cli/src/actions/shared/wrap_key.rs

```diff
@@ -87,7 +87,7 @@ impl WrapKeyAction {
             // Print the wrapping key for user. This is the only time that this wrapping key will be printed
             println!(
                 "Wrapping key: {}",
-                general_purpose::STANDARD.encode(key_bytes)
+                general_purpose::STANDARD.encode(key_bytes.deref())
```

Comment:
Your suggestion does not work as the Trait AsRef<[u8]> is not implemented for Secret.
I suggest to leave it like this instead of writing more code, especially since the end goal is the same ?